packages/components/tabs/README.mdx

```diff
@@ -71,7 +74,7 @@ Be aware that the users will not see all the content at the same time. Make sure
 ### used with TabPanel
 
 ```jsx
-<>
+<div>
   <Tabs role="tablist">
     <Tab id="first" selected={selected === 'first'}>
```

Comment:
just realized that this example doesn't work because `selected` is not defined here. this PR would be a nice opportunity to make it work.

Reply:
yes, just noticed it too